From 2944a69adeb35f1715cbc8289f828c82ff5fdacd Mon Sep 17 00:00:00 2001
From: Neil Armstrong <narmstrong@baylibre.com>
Date: Fri, 12 Apr 2019 04:31:00 -0400
Subject: [PATCH 082/178] FROMGIT: media: MAINTAINERS: Update AO CEC with
 ao-cec-g12a driver

Update the MAINTAINERS entry with the new AO-CEC driver for G12A.

Signed-off-by: Neil Armstrong <narmstrong@baylibre.com>
Signed-off-by: Hans Verkuil <hverkuil-cisco@xs4all.nl>
Signed-off-by: Mauro Carvalho Chehab <mchehab+samsung@kernel.org>
(cherry picked from commit a4dfc8a24796ff312d1d307e11f26f8ca466e938
 git://linuxtv.org/media_tree.git master)
---
 MAINTAINERS | 1 +
 1 file changed, 1 insertion(+)

diff --git a/MAINTAINERS b/MAINTAINERS
index 2c2fce7..ddc31a0 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -10057,6 +10057,7 @@ L:	linux-amlogic@lists.infradead.org
 W:	http://linux-meson.com/
 S:	Supported
 F:	drivers/media/platform/meson/ao-cec.c
+F:	drivers/media/platform/meson/ao-cec-g12a.c
 F:	Documentation/devicetree/bindings/media/meson-ao-cec.txt
 T:	git git://linuxtv.org/media_tree.git
 
-- 
2.7.1

